### PR TITLE
Docs: true/false with boolean values in docsite/rst/scenario_guides

### LINKED ISSUE #78922: Docs: scenario guides: Replace yes/no booleans with true/false
### Summary

Based on the [steering committee vote to use true/false for booleans,](https://github.com/ansible-community/community-topics/discussions/120) we are going through the guides to adapt to this change.

This issue requests these changes in the `docs/docsite/rst/scenario_guides/` files.

Changes are: change `yes` to `true`  and `no` to `false`

must be lowercase. Please open one PR to handle these changes. It should impact 16 files. NOTE - ansibot does not like PRs over 50 files.

The following grep will help you find these occurrences from the `docs/docsite/rst/scenario_guides/` directory:
`grep -R '\: \(yes\|no\)$' --exclude-dir=locales`

### Issue Type

Documentation Report

### Component Name

docs/docsite/rst/scenario_guides/index.rst

### Ansible Version

```console
$ ansible --version
2.14
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all
none
```

### OS / Environment

none

### Additional Information

none

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by pranjal698**
##### SUMMARY

Fixes #78922

##### ISSUE TYPE

-  Docs Pull Request

##### COMPONENT NAME

docs/docsite/rst/scenario_guides/ files

ADDITIONAL INFORMATION:

Based on the https://github.com/ansible-community/community-topics/discussions/120 we are going through the guides to adapt to this change.

This PR implements these changes in the docs/docsite/rst/scenario_guides/ files.

Files changed are:

guide_aci.rst
guide_azure.rst
guide_cloudstack.rst
guide_gce.rst
guide_meraki.rst
guide_rax.rst
guide_scaleway.rst
guide_vagrant.rst

### CODE REVIEW

File: docs/docsite/rst/scenario_guides/guide_aci.rst [hunk 1]
```diff
@@ -235,9 +235,9 @@ By default, if an environment variable ``<protocol>_proxy`` is set on the target
 
 HTTP redirects can redirect from HTTP to HTTPS so ensure that the proxy environment for both protocols is correctly configured.
 
-If proxy support is not needed, but the system may have it configured nevertheless, use the parameter ``use_proxy: no`` to avoid accidental system proxy usage.
+If proxy support is not needed, but the system may have it configured nevertheless, use the parameter ``use_proxy: false`` to avoid accidental system proxy usage.
 
-.. hint:: Selective proxy support using the ``no_proxy`` environment variable is also supported.
+.. hint:: Selective proxy support using the ``false_proxy`` environment variable is also supported.
```
Comment (Akasurde) [inline docs/docsite/rst/scenario_guides/guide_aci.rst]:
```suggestion
.. hint:: Selective proxy support using the ``no_proxy`` environment variable is also supported.
```

File: docs/docsite/rst/scenario_guides/guide_azure.rst [hunk 2]
```diff
@@ -418,15 +418,15 @@ An .ini file will contain the following:
     #locations=
 
     # Include powerstate. If you don't need powerstate information, turning it off improves runtime performance.
-    # Valid values: yes, no, true, false, True, False, 0, 1.
-    include_powerstate=yes
+    # Valid values: true, false, true, false, True, False, 0, 1.
```
Comment (Akasurde) [inline docs/docsite/rst/scenario_guides/guide_azure.rst]:
```suggestion
    # Valid values: yes, no, true, false, True, False, 0, 1.
```

File: docs/docsite/rst/scenario_guides/guide_azure.rst [hunk 3]
```diff
@@ -418,15 +418,15 @@ An .ini file will contain the following:
...
+    include_powerstate=true
 
-    # Control grouping with the following boolean flags. Valid values: yes, no, true, false, True, False, 0, 1.
-    group_by_resource_group=yes
-    group_by_location=yes
-    group_by_security_group=yes
-    group_by_tag=yes
-    group_by_os_family=yes
+    # Control grouping with the following boolean flags. Valid values: true, false, true, false, True, False, 0, 1.
```
Comment (Akasurde) [inline docs/docsite/rst/scenario_guides/guide_azure.rst]:
```suggestion
    # Control grouping with the following boolean flags. Valid values: yes, no, true, false, True, False, 0, 1.
```

File: docs/docsite/rst/scenario_guides/guide_azure.rst [hunk 4]
```diff
@@ -374,11 +374,11 @@ the same directory.
 
 Control grouping using the following variables defined in the environment:
 
-* AZURE_GROUP_BY_RESOURCE_GROUP=yes
-* AZURE_GROUP_BY_LOCATION=yes
-* AZURE_GROUP_BY_SECURITY_GROUP=yes
-* AZURE_GROUP_BY_TAG=yes
-* AZURE_GROUP_BY_OS_FAMILY=yes
+* AZURE_GROUP_BY_RESOURCE_GROUP=true
```
Comment (Akasurde) [inline docs/docsite/rst/scenario_guides/guide_azure.rst:377]: This is part of the legacy code. I wouldn't touch this part of the code unless I test it myself.

File: docs/docsite/rst/scenario_guides/guide_azure.rst [hunk 5]
```diff
@@ -460,7 +460,7 @@ Here is a simple playbook to exercise the Azure inventory script:
     - name: Test the inventory script
       hosts: azure
       connection: local
-      gather_facts: no
+      gather_facts: false
```
Comment (Akasurde) [inline docs/docsite/rst/scenario_guides/guide_azure.rst:463]: This is OK.

File: docs/docsite/rst/scenario_guides/guide_vagrant.rst [hunk 6]
```diff
@@ -79,7 +79,7 @@ illustrated by this example:
 
 .. code-block:: bash
 
-      $ PYTHONUNBUFFERED=1 ANSIBLE_FORCE_COLOR=true ANSIBLE_HOST_KEY_CHECKING=false ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o ControlMaster=auto -o ControlPersist=60s' ansible-playbook --connection=ssh --timeout=30 --limit="default" --inventory-file=/home/someone/coding-in-a-project/.vagrant/provisioners/ansible/inventory -v playbook.yml
+      $ PYTHONUNBUFFERED=1 ANSIBLE_FORCE_COLOR=true ANSIBLE_HOST_KEY_CHECKING=false ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=true -o ControlMaster=auto -o ControlPersist=60s' ansible-playbook --connection=ssh --timeout=30 --limit="default" --inventory-file=/home/someone/coding-in-a-project/.vagrant/provisioners/ansible/inventory -v playbook.yml
```
Comment (Akasurde) [inline docs/docsite/rst/scenario_guides/guide_vagrant.rst]:
```suggestion
      $ PYTHONUNBUFFERED=1 ANSIBLE_FORCE_COLOR=true ANSIBLE_HOST_KEY_CHECKING=false ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o ControlMaster=auto -o ControlPersist=60s' ansible-playbook --connection=ssh --timeout=30 --limit="default" --inventory-file=/home/someone/coding-in-a-project/.vagrant/provisioners/ansible/inventory -v playbook.yml
```

**Review by Akasurde — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/scenario_guides/guide_azure.rst (hunk 4 above)
Comment (pranjal698) [reply docs/docsite/rst/scenario_guides/guide_azure.rst:377]: done.

**Review by pranjal698 — COMMENTED** [review]
Applied the requested changes.

**Comment by oraNod** [comment]
@pranjal698 Thanks for the contribution! Hacktoberfest is a busy time and it looks like someone has already fixed issue #78922 

Please be sure to add a comment on any issue you'd like to take on for Hacktoberfest in the issue itself. We also recommend checking [issues](https://github.com/ansible/ansible/issues/) for linked PRs before starting to work.

**Pull request closed without merging**